detox/src/utils/regexp.js

```diff
@@ -0,0 +1,5 @@
+function isRegExp(obj) {
+  return Object.prototype.toString.call(obj) === '[object RegExp]';
```

Comment:
I assume this is because of the different sandboxes (Jest), right?

Reply:
Yes, simply checking `instanceof` fails in this case, so I had to use a workaround